src/common/meta/src/lib.rs

```diff
@@ -40,7 +40,7 @@ pub mod region_keeper;
 pub mod rpc;
 pub mod sequence;
 pub mod state_store;
-pub mod table_name;
+pub use table::table_name;
```

Comment:
Why re-export this

Reply:
Because the `table_name` was in `common-meta` before, and I moved it to `table` crate in this patch. Re-export it to avoid breaking too many codes.

Reply:
But let's break.